pyswarms/__init__.py

```diff
@@ -16,5 +16,8 @@
 
 from .single import global_best, local_best, general_optimizer
 from .discrete import binary
+from .utils.decorators import *
+from . import utils
```

Comment:
Hmmm, I think the previous version was enough where `cost` is defined explicitly.
I think the `@constraint` decorator in the future is better loaded whenever the user needs it.